tests/pillar/kibana_server.sls

```diff
@@ -9,4 +9,11 @@ kibana:
       engine: elasticsearch
       host: localhost
       port: 9200
-
+linux:
+  system:
+    enabled: true
+    repo:
+      kibana:
```

Comment:
This should not be default - remove completely or move into separate file (eg. kibana_debian_repo.sls)